code/datums/status_effects/debuffs.dm

```diff
@@ -136,3 +136,15 @@
 		owner.adjustBruteLoss(bleed_damage)
 	else
 		new /obj/effect/temp_visual/bleed(get_turf(owner))
+
+/datum/status_effect/pacifism
+	id = "pacifism_debuff"
+	alert_type = null
+	duration = 40 SECONDS
+
+/datum/status_effect/pacifism/on_apply()
+	ADD_TRAIT(owner, TRAIT_PACIFISM, id)
+	return ..()
+
+/datum/status_effect/pacifism/on_remove()
+	REMOVE_TRAIT(owner, TRAIT_PACIFISM, id)
```

Comment:
No parent return here?

Reply:
![image](https://user-images.githubusercontent.com/69320440/163726801-7239b6c6-c799-4732-ba6a-9ab33a5df152.png)
parent is an empty proc